protobuf/module_msg.proto

```diff
@@ -828,6 +828,9 @@ message ReplicateCommandSetGatesArg {
 message SetMetadataArg {
   /**
    * SetMetadata Attribute describes a metadata attribute and value to attach to every packet.
+   * If copying data from a packet buffer, SetMetadata will optionally shift
```

Comment:
Perhaps it would be a good idea to clarify that it is logical shift (not arithmetic shift) somewhere